src/constbuffer_array_tarray.c

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#include "c_pal/gballoc_hl.h"
+#include "c_pal/gballoc_hl_redirect.h"
+
+#include "c_util/tarray.h"
+#include "c_pal/thandle.h"
```

Comment:
>include "c_pal/thandle.h" [](http://example.com/codeflow?start=1&length=25)

Can we have the c_pal chilling with the other c_pal includes please?

Reply:
done